common/shlibs

```diff
@@ -3988,6 +3988,7 @@ libndr-samba-samba4.so samba-libs-4.13.2_1
 libndr-samba4.so samba-libs-4.13.2_1
 libprinter-driver-samba4.so samba-libs-4.13.3_1
 libsmb-transport-samba4.so samba-libs-4.13.2_1
+libnest2d_clipper_nlopt.so.1.0 libnest2d-0.4_1
```

Comment:
Nothing links to this library, so I think we can skip adding it here? If we can't, please don't break the samba block in half :)